### ISSUE TITLE
Relations

### ISSUE THREAD

**Issue by gmaclennan** [digidem/osm-p2p-db#5]
Relation members in OSM have `type` and `role` properties. How do we store that data in osm-p2p-db?

**Comment by substack** [comment]
This could be a top-level method.

**Comment by substack** [comment]
Right now, the metadata lives in the member documents themselves, which is slightly wrong. The breaking way to fix this is to force member records to be objects, not string ids.

**Comment by substack** [comment]
This is fixed in 3.4.0 and I've updated the main docs, architecture document, and osm-p2p docs:

```
osm.create({
  type: 'relation',
  members: [
    { type: 'relation', role: 'outer', ref: 'abc123' },
    { type: 'relation', role: 'inner', ref: '555fff' },
  ]
})
```
